<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBook</class>
 <widget class="QDialog" name="addBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a new Book</string>
  </property>
  <widget class="QLabel" name="Pagetitle">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Add a book to inventory</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>50</y>
     <width>391</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Vform">
    <item>
     <layout class="QHBoxLayout" name="Hform8">
      <item>
       <widget class="QLabel" name="isbn">
        <property name="text">
         <string>ISBN: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="isbnInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Hform7">
      <item>
       <widget class="QLabel" name="title">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Hform5">
      <item>
       <widget class="QLabel" name="author">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Hform4">
      <item>
       <widget class="QLabel" name="year">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yearInput">
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Hform3">
      <item>
       <widget class="QLabel" name="publisher">
        <property name="text">
         <string>Publisher:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="publisherInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Hform1">
      <item>
       <widget class="QLabel" name="genre">
        <property name="text">
         <string>Genre:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="genreInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Hform6">
      <item>
       <widget class="QLabel" name="price">
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="priceInput">
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Hform2">
      <item>
       <widget class="QLabel" name="quantity">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="quantityInput">
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addBookbutton">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>510</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addBookbutton</sender>
   <signal>clicked()</signal>
   <receiver>addBook</receiver>
   <slot>addNewBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNewBook()</slot>
 </slots>
</ui>
